docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '3.8'
+
+services:
+  mysql:
+    image: mysql
+    container_name: mysql
+    environment:  # Set environment variables for MySQL
+      MYSQL_ROOT_PASSWORD: root
+      MYSQL_DATABASE: usersystem
+      MYSQL_PASSWORD: root
+      MYSQL_TCP_PORT: 3307
+    ports:
+      - "3307:3307"  # port 3307 on the host to port 3307 on the container
+    volumes:
+      - mysql_data:/var/lib/mysql
+    networks:
+      - demo-network
+
+  springboot:
+    build: .  # Build the Spring Boot application
+    container_name: springboot
+    environment:  # Set environment variables for the Spring Boot application
+      SPRING_DATASOURCE_URL: jdbc:mysql://mysql:3307/usersystem?useSSL=false&serverTimezone=UTC&createDatabaseIfNotExist=true&allowPublicKeyRetrieval=true  # JDBC URL for the MySQL database
+      SPRING_DATASOURCE_USERNAME: root
+      SPRING_DATASOURCE_PASSWORD: root
```

Comment:
We should never push the password on GitHub.
Using the .env file can help solve this issues.